docs/docs/concepts/01-intro.md

```diff
@@ -13,9 +13,9 @@ between different parts of the AICA software stack.
 
 <SystemOverview className="themedSVG" style={{width: "100%"}}/>
 
-## AICA framework
+## AICA Core
 
-The AICA **framework** is a virtual robotics workspace pre-configured with a growing collection of software libraries
+**AICA Core** is a virtual robotics workspace pre-configured with a growing collection of software libraries
```

Comment:
```suggestion
The **AICA Core** is a virtual robotics workspace pre-configured with a growing collection of software libraries
```

Reply:
I see you removed the `The` everywhere, how would you argue it's better without?

Reply:
It is a stylistic choice, but generally I believe product names should be able to stand alone as a proper noun/ name without an article as opposed to being "A__" or "the__", which helps them be clearer and more memorable/ feel less generic. What are your thoughts here?

Reply:
I will not correct either of you guys in English language :smile:

Reply:
I don't want to be as strict as Apple with treating "iPhone" like a proper noun, so sometimes "the core" and "the studio" are still acceptable. But "AICA Core" and "AICA Studio" when used properly don't need a definite article (no need for the world "the" to be in front)